test/dash-playlist-loader.test.js

```diff
@@ -1102,7 +1102,7 @@ QUnit.test('starts without any metadata', function(assert) {
 });
 
 QUnit.test('moves to HAVE_MASTER after loading a master playlist', function(assert) {
-  const loader = new DashPlaylistLoader('dash.mpd', this.fakeHls);
+  const loader = new DashPlaylistLoader('dash.mpd', this.fakeHls, { withCredentials: false });
```

Comment:
why was this needed?

Reply:
there's also probably more tests now in this file that need to be converted from `false` to `{withCredentials: false}`

Reply:
I think the mock needed it because there was some assumption with options in the test code... somewhere.  I can try and remove it to see if it blows up.